tests/components/controlBar-test.js

```diff
@@ -934,14 +952,92 @@ describe('ControlBar', function () {
     expect(ccButtons.length).toBe(1);
   });
 
+  it('shows/hides multiaudio button if multiaudio available', function() {
+    var mockController = {
+      state: {
+        isMobile: false,
+        volumeState: {
+          volume: 1
+        },
+        closedCaptionOptions: {},
+        multiAudioOptions: {},
+        videoQualityOptions: {
+          availableBitrates: null
+        }
+      }
+    };
+
+    var oneButtonSkinConfig = Utils.clone(skinConfig);
```

Comment:
Please note that this creates a shallow copy, so if you modify any nested properties you will still be modifying the original object. This limitation is already addressed by `baseMockProps`, so that's another reason to switch to that.

Reply:
\+